### ISSUE TITLE
Transferring token by multisig raises "symbol precision mismatch" error

### ISSUE THREAD

**Issue by young-mu** [EOSIO/eos#5665]
I try to transfer token by multi-sign mechanism. I found that transferring with `quantity` argument `500 EOS`, instead of `500.0000 EOS`,  will raise `symbol precision mismatch` error.

Here are the details.

the permissions of test1 account are 

```shell
permissions:
     owner     1:    1 EOS7G6FMvusCAXyVBZyac1WuBMCuSDu5GcRv4kkPCspRcbyY6vAib
        active     1:    1 test2@owner,
```

So i want to propose a proposal to transfer token from test1. 

1. propose

```shell
cleos multisig propose transtest '[{"actor": "test2", "permission": "owner"}]' '[{"actor": "test1", "permission": "active"}]' eosio.token transfer '{"from": "test1", "to": "test2", "quantity": "50 EOS", "memo": "transfer test"}' -p test2@active
```

2. approve

```shell
cleos multisig approve test2 transtest '{"actor": "test2", "permission": "owner"}' -p test2@owner
```

3. exec

```shell
cleos multisig exec test2 transtest -p test2@active
```

The proposal failed with error logs of nodeos below and test2 fails to receive `50 EOS` token.

```shell
2018-09-14T10:16:31.006 thread-0   wasm_interface.cpp:929        eosio_assert         ] message: symbol precision mismatch
2018-09-14T10:16:31.007 thread-0   controller.cpp:688            push_scheduled_trans ] 3050003 eosio_assert_message_exception: eosio_assert_message assertion failure
assertion failure with message: symbol precision mismatch
    {"s":"symbol precision mismatch"}
    thread-0  wasm_interface.cpp:930 eosio_assert
pending console output:
    {"console":""}
    thread-0  apply_context.cpp:61 exec_one
```

But when I propose a new proposal again with `quantity` argument as `50.0000 EOS` instead of `50 EOS`, and then approve and execute this new proposal, it works well. (no error logs of nodeos and test2 does receive `50 EOS` token)

**Comment by damianoazzolini** [comment]
I think you should specify the same precision specified during the creation of the token. For instance, if the token has been defined with four decimal places (`X.0000`), every time you call an action, you should specify the same number of decimal places (`X.0000`). Fore more infos, you could have a look to `symbol.hpp` and `asset.hpp` files.

**Comment by young-mu** [comment]
Okay. 

Just as what you said, when I created the token like "100 EOS", and I passed the argument "50 EOS" (same precision) for transfer proposal, it does works.

---

Just come back to the initial issue, I created the token as "100.0000 EOS" and transferred it by multisig failed with the quantity argument "50 EOS". But I wonder that why I use the command to transfer, it can work.

```shell
cleos transfer eosio.token test1 "50 EOS"
```

(The quantity argument is also **"50 EOS"**)
